sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -449,4 +449,25 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
         .add(noteEntity);
     });
   }
+
+  /**
+   * Check if a program has at least one offering.
+   * @param programId program id.
+   * @returns true, if a program has existing offering.
+   */
+  async hasExistingOffering(
+    programId: number,
+    locationId: number,
+  ): Promise<boolean> {
+    const found = await this.repo
```

Comment:
the variable name can be more meaningful to the context(e.g offeringExist, offeringFound)